app/Importer/Importer.php

```diff
@@ -146,7 +146,7 @@ public function __construct($file)
      */
     public function import()
     {
-        $headerRow = $this->csv->fetchOne();
+        $headerRow = $this->csv->nth(0);
```

Comment:
I really hate that the CSV people make us do this - but that's not your fault. I get the same deprecation warning in PHPStorm when I look at this line, so this is probably the right thing to do. Sigh.